packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@mixin callout-data {
+  .#{$prefix}--callout {
```

Comment:
@RaphaelAmadeu this appears to be affecting other callout patterns. Can you update the class name to be specific to this pattern?